### DISCUSSION TITLE
Rodauth 2.26.1 Released

### DISCUSSION THREAD

**Discussion by jeremyevans** [jeremyevans/rodauth#281] [General]
Rodauth 2.26.1 has been released!

This release fixes a regression in 2.26.0 that caused QR codes used
in the otp feature to be all black rectangles.

Thanks,
Jeremy

**Comment by loranger32** [comment]
Thanks a lot for the fix !

Just a quick note if it can be useful for others : in my case, upgrading from Rodauth 2.26.0 to Rodauth 2.26.1 didn't fix the issue, I also needed to upgrade the Rqrccode gem from 2.1.1 to 2.1.2 to avoid black QR code.

Seems it has to do with [this commit in the Rqrcode repo](https://github.com/whomwah/rqrcode/commit/469651e4dcf061594627b250de6657617379dedc)

Anyway, now it's working fine, thanks again.
